frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/LibraryBody.module.css

```
@@ -1,9 +1,9 @@
 .libraryContent {
   display: grid;
-  grid-template-columns: 2fr 7fr 3fr;
+  grid-template-columns: 2fr 10fr;
```

Småpirk: Bruk så små enheter som mulig. Dette gir samme resultat:
```suggestion
  grid-template-columns: 1fr 5fr;
```

---

Såklart - bra sett!